valohai.yaml

```diff
@@ -0,0 +1,46 @@
+- step:
+    name: redshift-query
+    image: python:3.11-slim
+    command:
+    - pip install -r /valohai/repository/connectors/steps/redshift/requirements.txt -q
+    - PYTHONPATH=/valohai/repository python /valohai/repository/connectors/steps/redshift/query.py {parameters}
+    icon: https://upload.wikimedia.org/wikipedia/commons/7/73/Amazon-Redshift-Logo.svg
+    category: connectors
+    parameters:
+    - name: SQLquery
+      type: string
+      default: SELECT * FROM table
+      description: SQL query to be executed
+      widget: SQL
+    - name: datum_alias
+      type: string
+      description: Valohai datum alias given for the output
+      optional: true
+      widget: datumalias
+    - name: output_path
+      type: string
+      description: The name and path for the output file
+      default: results.csv
+    environment-variables:
+    - name: REDSHIFT_HOST
+      description: Redshift cluster endpoint url, e.g. "<cluster-identifier>.xxxxxxxxx.xx-xxxx-x.redshift.amazonaws.com"
+    - name: REDSHIFT_DATABASE
+      description: Database name to query from
+    - name: REDSHIFT_USER
+      description: Database user name
+    - name: REDSHIFT_REGION
+      description: AWS region of the Redshift cluster e.g. "eu-west-1"
+      default: "eu-west-1"
+    - name: REDSHIFT_PORT
+      description: The network port of the Redshift cluster endpoint
+      default: "5439"
+    - name: REDSHIFT_PASSWORD
+      description: Database user password. Only used if REDSHIFT_USE_IAM is false.
+      optional: true
```

Comment:
Amazon uses `RSPASSWORD` key on Amazon Redshift RSQL client, should we consider using the same name? https://docs.aws.amazon.com/redshift/latest/mgmt/rsql-query-tool-environment-variables.html

Reply:
Changed